runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -541,4 +642,89 @@ private static HiscoreEndpoint toEndPoint(final AccountType accountType)
 		private final String name;
 		private final HiscoreEndpoint endpoint;
 	}
+
+	private static String longBossName(String boss)
+	{
+		switch (boss.toLowerCase())
+		{
+			case "corp":
+				return "Corporeal Beast";
+
+			case "jad":
+				return "TzTok-Jad";
+
+			case "kq":
+				return "Kalphite Queen";
+
+			case "chaos ele":
+				return "Chaos Elemental";
+
+			case "dusk":
+			case "dawn":
+			case "galio":
+			case "gargs":
+				return "Grotesque Guardians";
+
+			case "archaeologist":
+				return "Crazy Archaeologist";
+
+			case "mole":
+				return "Giant Mole";
+
+			case "vetion":
+				return "Vet'ion";
+
+			case "kbd":
+				return "King Black Dragon";
+
+			case "sire":
+				return "Abyssal Sire";
+
+			case "smoke devil":
+				return "Thermonuclear Smoke Devil";
+
+			case "zuk":
+			case "inferno":
+				return "TzKal-Zuk";
+
+			// gwd
+			case "sara":
+			case "saradomin":
+			case "zilyana":
+				return "Commander Zilyana";
+			case "zammy":
+			case "zamorak":
+			case "kril":
+			case "kril trutsaroth":
+				return "K'ril Tsutsaroth";
+			case "arma":
+			case "kree":
+			case "kreearra":
+				return "Kree'arra";
+			case "bando":
+			case "bandos":
+			case "graardor":
+				return "General Graardor";
+
+			// dks
+			case "supreme":
+				return "Dagannoth Supreme";
+			case "rex":
+				return "Dagannoth Rex";
+			case "prime":
+				return "Dagannoth Prime";
+
+			case "wt":
+				return "Wintertodt";
+			case "barrows":
+				return "Barrows Chests";
+
+			case "xeric":
+			case "olm":
```

Comment:
"raids" maybe too? also, probably add abbreviation "verzik" and "raids 2" for Theatre of Blood